package.json

```diff
@@ -58,7 +58,7 @@
       }
     },
     "start": {
-      "command": "babel-node bin/server",
+      "command": "npm run lint && babel-node bin/server",
```

Comment:
Hmm, looks like we have a difference in approach! 😄 

My preference is always for the developer to fix any lint issues immediately before running the app. They're usually small and easy to fix so it shouldn't ever be much of a block. Some issues can be caused by linter violations so it's sometimes not worth running the app if there's a linter error or warning.

Wouldn't the developer only run deploy once they're finished with development? At that stage it seems too late to get some of the benefits of the linter.